salesforce-pubsub/1.0/modules/ROOT/pages/salesforce-pubsub-connector-reference.adoc

```diff
@@ -56,7 +56,7 @@ Uses a username and a password for authentication.
 | Security Token a| String |  User's security token. The security token can be omitted if your IP has been added to the allowlist on Salesforce. |  |
 | Authorization URL a| String |  Web service URL responsible for user authentication. The URL for the endpoint that is configured to handle SOAP authentication requests. |  |
 | TLS configuration a| <<Tls>> |  Configures TLS. If using the HTTPS protocol, you must configure TLS.  |  |
-| gRPC host a| String |  Host in which the proxy requests are sent to the gRPC API (towards the gRPC host). The proxy requests to the gRPC host go through the proxy. |  `api.pilot.pubsub.salesforce.com` |
+| gRPC host a| String |  Host in which the proxy requests are sent to the gRPC API (towards the gRPC host). The proxy requests to the gRPC host go through the proxy. |  `api.pubsub.salesforce.com` |
```

Comment:
Is this the host from which the request are sent? If so, "Host from which..."

Is the second sentence needed?

Reply:
Nope, the requests are sent in the host. The second sentence is needed. We specifically added it to provide clarity since customers were confused. See https://github.com/mulesoft/docs-connectors/pull/2266/